microsoftgraph/microsoft-graph-explorer-v4
# Task: tenant name ellipsis

## Overview
Adds ellipsis to long tenant names and displays a tooltip with the full tenant name on hover

### Demo
<img width="284" alt="image" src="https://user-images.githubusercontent.com/45680252/171563210-49c36fb7-620b-427f-9ff8-c8bd10ae4114.png">


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Go to 'MainHeader.tsx' file and pass a long string to the renderLabel function. Pass the same string to the tooltip content prop
Reload and notice the ellipsis applied on the name.
Hover over the tenant label and see the tooltip with the full string
